synapse/cortex.py

```diff
@@ -3180,20 +3190,22 @@ async def addNodeTag(self, user, iden, tag, valu=(None, None)):
         '''
 
         buid = s_common.uhex(iden)
-        async with await self.snap(user=user) as snap:
+        node = await self.view.getNodeByBuid(buid)
```

Comment:
The telepath version of this function ( CoreApi.addNodeTag ) was deleted a while back in v2.156.0 and this function should have been deleted then.